packages/core/test/unit/sql/generateJoin.test.js

```diff
@@ -122,6 +122,7 @@ describe('QueryGenerator#generateJoin', () => {
       },
       {
         default: 'LEFT OUTER JOIN [company] AS [Company] ON [User].[company_id] = [Company].[id]',
+        oracle: `LEFT OUTER JOIN "company" "Company" ON "User"."company_id" = "Company"."id"`,
```

Comment:
This will affect more tests, but it might be worth updating `expectSql` to replace the 'AS' with '' so you do not need to define as much changed expectations. Something like;
```
  if (combinedExpectations.has(usedExpectationName) && typeof expectation === 'string') {
    // replace [...] with the proper quote character for the dialect
    // except for ARRAY[...]
    expectation = replaceGenericIdentifierQuotes(expectation, dialect);
    if (dialect.name === 'ibmi') {
      expectation = expectation.trim().replace(/;$/, '');
    }
    if (dialect.name === 'oracle') {
      expectation = expectation.replace('AS', '');
    }
  }
```